pom.xml

```diff
@@ -1,234 +1,225 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <groupId>org.jahia.modules</groupId>
-        <artifactId>jahia-modules</artifactId>
-        <version>8.1.0.0</version>
-    </parent>
-    <artifactId>sitemap</artifactId>
-    <version>5.0.0-SNAPSHOT</version>
-    <packaging>bundle</packaging>
-    <name>Jahia Sitemap</name>
-    <description>These are the custom module (Jahia Sitemap) for running on a Jahia server.</description>
-    <scm>
-        <connection>scm:git:git@github.com:Jahia/sitemap.git</connection>
-        <developerConnection>scm:git:git@github.com:Jahia/sitemap.git</developerConnection>
-        <url>scm:git:git@github.com:Jahia/sitemap.git</url>
-        <tag>HEAD</tag>
-    </scm>
-    <properties>
-        <require-capability>osgi.extender;filter:="(osgi.extender=org.jahia.bundles.blueprint.extender.config)"
-        </require-capability>
-        <jahia-depends>site-settings-seo=[3.0,99],graphql-dxm-provider</jahia-depends>
-        <jahia-module-signature>MCwCFA8mIQkUfH0hHWkqsEuMA1LWaWhYAhRoNUCxJznNeUn0LeXX1vz3/FOuYg==</jahia-module-signature>
-        <yarn.arguments>build:production</yarn.arguments>
-        <export-package>org.jahia.modules.sitemap.utils</export-package>
-        <sonar.sources>src/javascript</sonar.sources>
-        <import-package>
-            graphql.annotations.annotationTypes;version="[7.2,99)",
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
I think that you have an indent issue on the pom.xml file

Reply:
Indeed, updated!